src/index.ts

```diff
@@ -372,6 +373,15 @@ export enum SupportedOperatorEnum {
   WAVE = 'WAVE',
 }
 
+// Map of all supported fiat account schemas to the corresponding schema type. List must be manually updated
+export type FiatAccountSchemas = {
```

Comment:
I think exporting just this instead of all the individual schemas should be sufficient and also enforce better types in clients. Although this could mean breaking changes in multiple places in the wallet. Thoughts?

Reply:
Having this type here is better than the alternative (the wallet doing a very similar mapping on its own). I think its reasonable to force this behavior